privaterelay/locales/de/privaterelay/brands.ftl

```diff
@@ -0,0 +1,171 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### Please read the trademarks localization guide
+### https://mozilla-l10n.github.io/styleguides/mozilla_general/index.html#brands-copyright-and-trademark
```

Comment:
This directory (`privaterelay/locales/de/`) is temporary to verify Fluent works as expected. These files will be removed before merging.